packages/dx-react-grid/docs/reference/selection-state.md

```diff
@@ -30,7 +30,6 @@ availableToSelect | Getter | Array&lt;[Row](grid.md#row)&gt; | Rows to be render
 
 Name | Plugin | Type | Description
 -----|--------|------|------------
-setRowSelection | Action | ({ rowId }) => void | A function that selects a row
-setRowsSelection | Action | ({ rowIds }) => void | A function that selects multiple rows
+setRowsSelection | Action | ({ rowIds: Array&lt;number &#124; string&gt;, selected?: boolean  }) => void | A function that selects/deselects rows. To select/deselect a single row, use an array with a single item as the `rowIds` argument. The `selected` argument specifies whether selection should be added or removed
```

Comment:
Is it required to write a note about wrapping a single element? It seems obviously without a note.